packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/ExtractionService.java

```diff
@@ -440,7 +440,7 @@ private DocumentAsset runVariableExtraction(
 				for (final UUID modelId : modelIds) {
 					notificationInterface.sendMessage("Attempting to align models for model: " + modelId);
 					try {
-						alignAMR(documentId, modelId, hasWritePermission).get();
```

Comment:
This was running async and not included in the transaction, so the attributes in the metadata would not be visible.